### PR TITLE
soc/power9/istep_18_{11,12}.c: fix order and values of SCOM operations

### PR DESCRIPTION
**Pull request by krystian-hebel**
Signed-off-by: Krystian Hebel <krystian.hebel@3mdeb.com>
Change-Id: Ifec3198d08c24d673c861c3dcaddcc6661b2b9d2

### CODE REVIEW

**Comment by macpijan** [comment]
@IgorBagnucki please reivew

File: src/include/cpu/power/istep_18.h [hunk 1]
```diff
@@ -13,8 +16,10 @@ void istep_18_12(void);
 #define PERV_TOD_TX_TTYPE_REG_TRIGGER (0)
 
 #define MDMT_TOD_GRID_CYCLE_STAGING_DELAY (6)
-#define FREQ_X_MHZ (0x708)
+#define FREQ_X_MHZ (1800)
 #define TOD_GRID_PS (400)
+
+/* FIXME: P9A?! */
```
Comment (IgorBagnucki) [inline src/include/cpu/power/istep_18.h:22]: This value is correct. Compare here: https://git.raptorcs.com/git/talos-hostboot/tree/src/import/chips/p9/procedures/hwp/nest/p9_tod_setup.C#n793

File: src/soc/ibm/power9/istep_18_11.c [hunk 2]
```diff
@@ -3,85 +3,184 @@
...
+	/*
+	 * TODO: this touches XBUS/OBUS, but Hostboot doesn't modify registers so
+	 * skip it for now and fix if needed.
+	 */
+	//scom_and_or(PERV_TOD_PRI_PORT_0_CTRL_REG, ???, ???);
+	//scom_and_or(PERV_TOD_SEC_PORT_0_CTRL_REG, ???, ???);
+
+	/* Configure M_PATH_CTRL_REG */
+	/*
+	 * TODO: check this again. Value is correct, not sure whether fields are
```
Comment (IgorBagnucki) [inline src/soc/ibm/power9/istep_18_11.c:98]: Done a quick analysis of cleared fields. Looks correct to me.

**Review by IgorBagnucki — APPROVED** [review]
Looks good to me.

**Pull request merged**